themes/default/shop/config.json5

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
might want to add json5 to prettier config. Looks like it's doing 2 space indent here.

Reply:
76e3be6